src/components/LoansByCategory/QuickFilters/QuickFilters.vue

```diff
@@ -1,10 +1,24 @@
 <template>
 	<div class="tw-flex tw-flex-col tw-mb-2 tw-w-full">
+		<div v-if="!withCategories && !enableFilterPills" class="tw-flex tw-items-center tw-mb-2">
```

Comment:
Why were these two blocks of new elements needed? I thought the old version of the filters already were shown on LH.

Reply:
These were removed with ui update, so, if we need to keep design like here https://www.kiva.org/lend-by-category/agriculture for testing, we need these new(old) blocks, right?
Like here https://github.com/kiva/ui/blob/155c55d83dde1f5180c9964ec9130635a4d41fa0/src/components/LoansByCategory/QuickFilters/QuickFilters.vue#L2

Reply:
If this was how it was before, then makes sense, thanks.